dev-requirements.txt

```diff
@@ -7,7 +7,7 @@ vcrpy>=1.11.0,<=1.11.99
 vcrpy-unittest==0.1.6
 sphinx_rtd_theme>=0.2.4,<1.0.0
 numpydoc>=0.7.0,<1.0.0
-feather-format
+feather-format; sys_platform != 'win32' or python_version >= '3.5'
```

Comment:
`feather-format` depends on `pyarrow`, which doesn't have a Windows installer for Python < 3.5; see [here](https://arrow.apache.org/docs/python/install.html). Not having `feather-format` installed only [skips one test](https://github.com/civisanalytics/civis-python/blob/403f318294ab4e204fd1f0c4fb0678105fdbe1d7/civis/ml/tests/test_model.py#L180), which I think is fine (and we _are_ running this test for Python >= 3.5 on Windows anyway).